Tests/KituraTests/TestServer.swift

```diff
@@ -149,4 +150,57 @@ class TestServer: KituraTest {
             XCTFail("Unexpected error starting server: \(error)")
         }
     }
+
+    func testServerRestart() {
+        let port = httpPort
+        let path = "/testServerRestart"
+        let body = "Server is running."
+
+        let router = Router()
+        router.get(path) { _, response, next in
+            response.send(body)
+            next()
+        }
+        Kitura.addHTTPServer(onPort: port, with: router)
+
+        let server = Kitura.httpServersAndPorts[0].server
```

Comment:
Why not just write `let server = Kitura.addHTTPServer(onPort: port, with: router)` on the previous line?